### ISSUE TITLE
Missing code signing entitlements

### ISSUE THREAD

**Issue by qgliu** [aws/aws-sdk-ios#596]
I've implemented the AWS SDK, but in submitting the archive to iTunes connect I keep getting the error saying "Missing Code Signing Entitlements. No entitlements found in bundle 'com.amazonaws.sdk.ios.AWSResource'" for executable 'Payload/**.app/Frameworks/AWSLex.framework/AWSResources.bundle/AWSResources'

It also say "Invalid bundle structure".

I use version SDK 2.5.0, Xcode 8.2.1, Swift 3. I use Carthage to implement the SDK.

In using SDK, everything works very well. It's just in uploading the archive, I start to have problems.

Anyone can help me out on this? Thank you!

**Comment by mlshon** [comment]
What do you have in your resource bundle?
Do you have any executables?

**Comment by qgliu** [comment]
I checked the frameworks installed by Carthage as well as the frameworks that can be downloaded directly from aws website, there is a AWSResouces.bundle file appears in the AWSLex.framework but not in any other frameworks. Is this a mistake?

I don't really need AWSLex.framework, so was able to finally solve my problem by removing the framework from the embedded libraries. In this process I had another issue. In removing the framework it doesn't automatically removed it from the linked library. So the compiler was complaining the missing of the framework. If it is not that I found an answer on stack overflow, I wouldn't know that I could manually remove it in the Link Binary With Libraries.

This is a quite different experience comparing to the other frameworks in working with Carthage. Adding or removing other open-source frameworks in Carthage is a bit more straightforward.

Thanks for your reply and I am so relieved that everything finally works.
